### PR TITLE
Replaced shell script for CentOS 7

### PR DESCRIPTION
**Pull request by anithamuthu79**
## High-level description

This is to update the CentOS 7 shell script for 1.10. 
https://jira.mesosphere.com/browse/DOCS-2454

### CODE REVIEW

**Comment by gauripowale** [comment]
@mesosphere-mergebot bump-ee

**Comment by gauripowale** [comment]
@mesosphere-mergebot bump-ee

**Comment by gauripowale** [comment]
@mesosphere-mergebot bump-ee

**Comment by gauripowale** [comment]
@mesosphere-mergebot label ready for review

**Comment by anithamuthu79** [comment]
@mesosphere-mergebot label ready for review

File: cloud_images/centos7/install_prereqs.sh [hunk 1]
```diff
@@ -50,10 +94,10 @@ echo ">>> Removing tty requirement for sudo"
 sed -i'' -E 's/^(Defaults.*requiretty)/#\1/' /etc/sudoers
 
 echo ">>> Install Docker"
-curl -fLsSv --retry 20 -Y 100000 -y 60 -o /tmp/docker-engine-1.13.1-1.el7.centos.x86_64.rpm \
-  https://yum.dockerproject.org/repo/main/centos/7/Packages/docker-engine-1.13.1-1.el7.centos.x86_64.rpm
-curl -fLsSv --retry 20 -Y 100000 -y 60 -o /tmp/docker-engine-selinux-1.13.1-1.el7.centos.noarch.rpm \
-  https://yum.dockerproject.org/repo/main/centos/7/Packages/docker-engine-selinux-1.13.1-1.el7.centos.noarch.rpm
+curl -fLsSv --retry 20 -Y 100000 -y 60 -o /tmp/docker-engine-1.12.6-1.el7.centos.x86_64.rpm \
+  https://yum.dockerproject.org/repo/main/centos/7/Packages/docker-engine-1.12.6-1.el7.centos.x86_64.rpm
```
Comment (sschneid) [inline cloud_images/centos7/install_prereqs.sh]: https://docs.mesosphere.com/1.10/installing/oss/custom/system-requirements/install-docker-centos/ states `1.13.1`; why are we reverting?

**Comment by sschneid** [comment]
https://jira.mesosphere.com/browse/DOCS-2454 is tagged with and mentions `1.11`, but this PR is targeted for `1.10`.  In additional, Docker installation is now in a separate script: https://github.com/dcos/dcos/blob/master/cloud_images/lib/el7/install_docker.sh

**Comment by orsenthil** [comment]
@mesosphere-mergebot label holding

**Pull request closed without merging**